interview-questions.md

```diff
@@ -8,36 +8,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
-action: "GET" location: /students/2
+action: "GET" location: /students/2 = def show - most likely will have a .find(params[:id]) and will take you to view a page with student id:2 information.
 
-action: "GET" location: /students/2/edit
+action: "GET" location: /students/2/edit = def edit - will take you to a view page for editing student with id:2's information. (This is a put action I believe) 
 
-action: "PATCH" location: /students/2
+action: "PATCH" location: /students/2 = def update - this will allow you to submit updated changes from editing student with id:2's information
 
-action: "DELETE" location: /students/2
+action: "DELETE" location: /students/2 = def destroy - Will delete the information of student at id:2
 
 5. As a developer, you are making an application to manage your to do list. Create 10 user stories that will help you get your application started. Read more about [user stories](https://www.atlassian.com/agile/project-management/user-stories).
+
+1 - As a CEO of a new start-up company for finding online deals, I want  an easy to use GUI, tso that it allows our members to easily navigate our application and find the deals that fit their shopping needs.
+
+2 - As a lawfirm manager, I would like our companies online filing system to be dynamic allowing us to store thousands of files for our clients in specific and easily findable folders in a cloud based application.
+
+3 - As a Veteran Agency, I would like to have a website friendly to senior citizens, so that they can get updates about our benefits programs more conveniently.
+
+4 - As Tom, I would like to be able to be able to upload ideas for my book into a workspace that can be accessible anywhere so I can continue where I stopped while on the go.
+
+5- As a travel agent I would like to be able to compare multiple rates together at a single time, so that I can find the best deals for my clients.
+
+6 - As a professional photographer I would like to be able to arrange many different portfolios of my work within a single app, so that I can customize them for different potential clients
+
+7 - As a musician I would like to be able to make changes to songs I am working on from my phone, so that when I am not able to work in the studio I can still get some work done.
+
+8- As a soda company CEO I would like to be able to organize the recipe for different flavors into a safe dynamic file sharing system, so that I can pass updates to my co-workers
+
+9- As a school teacher I would like to be able to show parents how well their children did in class today from an online forum, so that we can reach out to more than one parent at a time and need less face to face encounters.
+
+10 - As a developer I want to learn the most effecient tactics in coding, so that I can code with ease and show potential employers my skills.
```

Comment:
Love the format on these! The goal with user stories for our purposes is to be specific so that we have an idea of what we need. For example, with a to-do list, one user story could be something like As a user, I can create a new task. This will help us plan out how we want to do that ie controller, view, route for a new task.